src/migrators/index.js

```diff
@@ -0,0 +1,8 @@
+module.exports = async (strapi) => {
+  const config = strapi.config.get("migrations");
+  if (config?.enabled === true && config?.migrators?.length > 0) {
+    for (const migrator of config.migrators) {
+      await require(`./${migrator}`)(strapi);
```

Comment:
TIL you can do this!

Reply:
that's indeed sophisticated, I've seen it before but didn't know you can do dynamic value within the parenthesis.